files/en-us/mdn/structures/page_types/html_element_page_template/index.html

```diff
@@ -5,51 +5,70 @@
   - Element
   - HTML
   - Template
-  - reference page
+  - Reference
 browser-compat: path.to.feature.NameOfTheElement
 ---
 <div>{{MDNSidebar}}</div>
 
-<div class="note">
-<h2 id="Remove_before_publishing">Remove before publishing</h2>
 
-<h3 id="Title_and_slug">Title and slug</h3>
+<h3 id="Page_front_matter">Page front matter</h3>
 
-<p>An HTML element page should have a <em>title</em> of <em>"&lt;" + Name</em>Of<em>TheElement + "&gt;"</em> + <em>Description of element's purpose</em>. For example, the <code><a href="/en-US/docs/Web/HTML/Element/video">&lt;video&gt;</a></code> element has a <em>title</em> of <em>&lt;video&gt;: The Video Embed element</em>.</p>
+<p>The frontmatter at the top of the page is used to define "page metadata". The values should be updated appropriately for the particular element.</p>
```

Comment:
Not sure why it would need to be "page metadata".
```suggestion
<p>The frontmatter at the top of the page is used to define page metadata. The values should be updated appropriately for the particular element.</p>
```